extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/indexing-configuration.xml

```diff
@@ -127,8 +127,8 @@
         <properties-param>
           <name>constructor.params</name>
           <property name="index_alias" value="profile_alias"/>
-          <property name="index_previous" value="profile"/>
-          <property name="index_current" value="profile_v2"/>
+          <property name="index_previous" value="profile_v2"/>
+          <property name="index_current" value="profile_v3"/>
```

Comment:
Let's avoid this, when a user has an updat on his profile, the profile will be indexed automatically on user login. All active users will have their profiles reidexed for 6.2

Reply:
Please makje sure to announce this so that it is considered when the fix is delivered.

Reply:
done